### DISCUSSION TITLE
[Feature] Synology Photos Unsynchronized Change Feature

### DISCUSSION THREAD

**Discussion by toorux** [immich-app/immich#12192] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I hope the app can add a 'Unsynced Changes' feature similar to Synology Photos. In Synology Photos, after I delete photos on the web, the app shows an option for unsynced changes, allowing me to easily delete the photos that were removed from the cloud but not from the phone. This would prevent the deleted photos from being re-synced to the cloud after clearing the trash, rather than them being re-uploaded from the phone.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#3594
